quantum/keymap.h

```diff
@@ -40,6 +40,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #if defined(PROTOCOL_CHIBIOS)
 #    define RESET QK_RESET
 #endif
+// Gross hack, remove me and change RESET keycode to QK_BOOT
+#if defined(__AVR_AT90USB647__) || defined(__AVR_AT90USB1287__)
+#    undef RESET
+#endif
```

Comment:
@drashna what do you think about this? :P
`donutcables/scrabblepad:default` now compiles with MCU set to 1287.

Reply:
I'm really not a fan of this. but short of this or something similar, for now ...  I'm not sure what else we can really do.

Reply:
Yeah, but at least it's right next to a similar hack for ChibiOS, so when we eventually get round to renaming `RESET`, we can take them both out in one fell swoop.